### PR TITLE
Suggest solid:nvmQuota property (addresses #29)

### PR DESCRIPTION
**Pull request by kjetilk**
I'd like to explore this way of doing it in code.

### CODE REVIEW

**Comment by kjetilk** [comment]
OK, yeah, we can do that.

File: solid-terms.ttl [hunk 1]
```diff
@@ -195,3 +195,10 @@
     rdfs:label "where"@en ;
     rdfs:domain :Patch ;
     rdfs:range log:Formula .
+
+:storageQuota
+    a rdf:Property, owl:DatatypeProperty ;
```
Comment (RubenVerborgh) [inline solid-terms.ttl]: Domain and range?

Comment (kjetilk) [reply solid-terms.ttl]: Domain is `solid:Account` and/or `pim:Storage`. Range could be `byte-unit:Unit`, or just `rdfs:Literal`. I'm not sure any would look for that. Should we have them in?

Comment (kjetilk) [reply solid-terms.ttl]: Added `solid:Account` as domain for now, we can do range later once we find a home for #35 .

**Pull request merged**